app/frontend/components/pages/dashboard/walletOperationStatus.tsx

```diff
@@ -1,20 +1,41 @@
-import {h, Fragment} from 'preact'
+import {Fragment, h, JSX} from 'preact'
+import {useState, useEffect} from 'preact/hooks'
+
 import {useActions, useSelector} from '../../../helpers/connect'
 import actions from '../../../actions'
 import assertUnreachable from '../../../helpers/assertUnreachable'
 
-const PinkSpinner = ({message}) => (
-  <div className="header-message">
-    {message.split('\n').map((line, i) => (
-      <p className="loading-message black" key={i}>
-        {line}
-      </p>
-    ))}
-    <div className="spinner pink">
-      <span />
+const DelayRendering = ({children}) => {
```

Comment:
Rather then creating condition whitin `PinkSpinner` `useEffect` I've created new component for delay rendering, Seems more elegant imho